fossology/fossology
# feat(phpunit): Migrated to namespaced phpunit.

## Description

Add forward compatibility for the following change in PHPUnit 6
> PHPUnit's units of code are now namespaced. For instance, `PHPUnit_Framework_TestCase` is now `PHPUnit\Framework\TestCase

by implementing

> Added phpunit\framework\TestCase as an alias for PHPUnit_Framework_TestCase for forward compatibility

See: https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-6.0.0
See: https://github.com/sebastianbergmann/phpunit/blob/5.4/ChangeLog-5.4.md

## How to test

Run all tests or check the logs on travis-ci.

Thanks to @GMishx for pointing out this backwards compatibility issue in #1116
